functors task (1)/student/include/impl.h

```diff
@@ -40,9 +47,17 @@ struct MixingPolicy
      * Whiskey + SevenUp = SevenPlusSeven;
      * Others + Others = Oops;
      */
-    static Cocktail mix(/**???*/)
+    static Cocktail mix(AlcoholDrink alco, NonAlcoholDrink nonAlco)
     {
+		Cocktail result = Cocktail::Oops;
+		if (alco == AlcoholDrink::Gin && nonAlco == NonAlcoholDrink::LimeJuice)
```

Comment:
consider switch here
also, follow this rule: https://wiki.c2.com/?AlwaysUseBracesOnIfThen